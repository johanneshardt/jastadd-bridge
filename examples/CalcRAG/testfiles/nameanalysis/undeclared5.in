let
	you = let
		me = you
	in
		me
	end
in
	you
end
