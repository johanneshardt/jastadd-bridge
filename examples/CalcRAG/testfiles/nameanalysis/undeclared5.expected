Error at line 3: the definition of symbol 'you' is circular
